Makefile

```diff
@@ -245,7 +245,6 @@ test-run:
 .PHONY: test-all
 test-all:
 	@$(PYTHON3) -m pytest \
-			$(SRCDIR)/test \
```

Comment:
Why not change it to just `$(SRCDIR)`? Recently we faced strange issue where pytest called outside of osbuild dir was failing, but called from the inside of the repo was working fine.

Reply:
I can add it of course - it just seems to be redundant AFAICT, i.e. --rootdir is already setting things in the expected way. And `make -C /path/to/osbuild` also works. But maybe I'm missing something? Anything specific you have in mind?

Reply:
True, I forgot that `--rootdir` will do the trick:)